starter/source/interfaces/int25/i25remlin.F

```diff
@@ -584,7 +584,7 @@ SUBROUTINE REMN_I2OP_EDG25(N         ,FLAGREMNODE, IPARI   ,INTBUF_TAB   ,I2NODE
             ENDIF
 C  
                                
-            IF(JJJ + MAX_INSERTED_EDG > SIZE_INSERTED_EDG) THEN
+            IF(JJJ + MAX_INSERTED_EDG*MAX_INSERTED_I2  > SIZE_INSERTED_EDG) THEN
 C extend INSERTED_EDG if needed
                OLDSIZE = SIZE_INSERTED_EDG
                SIZE_INSERTED_EDG = SIZE_INSERTED_EDG  + MAX(NEDGE,MAX_INSERTED_EDG)
```

Comment:
Should it be `MAX(NEDGE,MAX_INSERTED_EDG*MAX_INSERTED_I2)` instead of `MAX(NEDGE,MAX_INSERTED_EDG)` ?